youtube_dl/extractor/canvas.py

```diff
@@ -53,6 +53,15 @@ def _real_extract(self, url):
                     'url': format_url,
                 })
         self._sort_formats(formats)
+        
+        subtitles = {}
+        for target in data['subtitleUrls']:
```

Comment:
Seems unlikely but reasonable, I'll update and squash the commits tomorrow ( it's 5am, I'm off to bed )
Thanks reviewing the code